portfolio/src/main/webapp/style.css

```diff
@@ -124,15 +129,16 @@ body {
 }
 
 .intro-wrapper h3 {
-  font: lighter italic "Roboto Light";
-  margin:0;
+  font: italic 2rem Roboto Light;
```

Comment:
The style guide suggests using single quotes to enclose font names with spaces.

Reply:
Thanks, rectified this in commit 7e0f486